tools/accuracy_checker/accuracy_checker/annotation_converters/README.md

```diff
@@ -379,7 +379,10 @@ The main difference between this converter and `super_resolution` in data organi
   * `formula_file` - path to file containing one formula per line
   * `split_file` - path to file containing `img_name` and corresponding formula `index` in `formula_file` separated by tab per line
   * `vocab_file` - file containing vocabulary to cast token class indices into human-readable tokens 
-
+* `cocosnet` - converts ADE20K dataset to `CocosnetAnnotation` according to `reference_dict`.
```

Comment:
I think, we need to get more common name for converter. There are other models for image to image translation which also can use this dataset e. g. https://github.com/NVlabs/SPADE
 
We prefer convention <dataset_name>_<task> (for example, ade20k_image_translation or ade20k_style_transfer)